pymc_marketing/mmm/base.py

```diff
@@ -467,9 +467,6 @@ def plot_components_contributions(self, **plt_kwargs: Any) -> plt.Figure:
         return fig
 
     def plot_channel_parameter(self, param_name: str, **plt_kwargs: Any) -> plt.Figure:
-        if param_name not in ["alpha", "lam", "beta_channel"]:
-            raise ValueError(f"Invalid parameter name: {param_name}")
-
```

Comment:
How will we handle this, then? For generic adstock and saturation functions?

Reply:
We will have to make a check based on the values of 
```python
saturation: SaturationTransformation = self.saturation
if param_name not in list(saturation.variable_mapping.values()): 
    raise ValueError(f"Invalid parameter name: {param_name}")

Reply:
Do you plan to remove this function? I think we should keep it as people do use it